### ISSUE TITLE
Focal Fossa crashing device upon opening a program

### ISSUE THREAD

**Issue by ShipmasterKyle** [dnschneid/crouton#4702]
```
name: focal
encrypted: no
Entering /media/removable/USB Drive/chroots/chroots/focal...
crouton: version 1-20220111101058~master:a9eafaee
release: focal
architecture: amd64
xmethod: xorg
targets: xorg,audio,core,gtk-extra,x11,xfce
host: version 14388.61.0 (Official Build) stable-channel grunt 
kernel: Linux localhost 5.10.83-13632-g02eb545eaaaf #1 SMP PREEMPT Tue Feb 15 19:36:52 PST 2022 x86_64 x86_64 x86_64 GNU/Linux
freon: yes

```
#### Please describe your issue:
A few weeks ago I installed crouton on a slim flash drive to save space. After a ton of issues and factory resetting my chromebook (HP 14) I was finally able to open my environment again. However upon starting any program such as Google or VSC my screen goes black and then turns on again.

#### If known, describe the steps to reproduce the issue:
Mount the flash
```
sudo mount -o remount,symfollow /media/removable/USB\ Drive/
```
Start the chroot environment from the flash drive
```
sudo startxce4 -c /media/removable/USB\ Drive/chroots/chroots
```
Run any program.

**Comment by TheDacoMelon** [comment]
Hi, this seems quite similar to something i was experiencing. Did Crouton work fine when you were on specifically the 4.14 kernel (Chrome OS like, version ~94)? Because any kernel version beyond that also forced my system to reboot when running apps, specifically GUI ones. I believe the GPU has problems, quite sure it is an issue with AMD. I am not exactly smart though so i cannot help beyond telling you this. You may try rolling back to version 94 and trying Crouton but google will forcefully update your OS to a newer version and at the same time newer and dysfunctional kernel. Good luck!

**Comment by ShipmasterKyle** [comment]
> Hi, this seems quite similar to something i was experiencing. Did Crouton work fine when you were on specifically the 4.14 kernel (Chrome OS like, version ~94)? Because any kernel version beyond that also forced my system to reboot when running apps, specifically GUI ones. I believe the GPU has problems, quite sure it is an issue with AMD. I am not exactly smart though so i cannot help beyond telling you this. You may try rolling back to version 94 and trying Crouton but google will forcefully update your OS to a newer version and at the same time newer and dysfunctional kernel. Good luck!

Yeah ChromeOS 95 and up have been having major issues with crouton. 98 was especially bad. I'll look into that solution.

**Comment by TomTravis** [comment]
yes google chrome does have some issues with arm64 not have the experience with AMD64 x86_64 machines
the arm64 would reset the machine on startup of chrome apps and create a dirty FS 
i now use Firefox this seem to do well
my experience with cross dev arm64 on amd64 has not be good for chromium
i using chrome 107 now 
the crash handler Cwr i believe fails to load now
i use ptrace this ususally  shows nothing
